contracts/HATTimelockController.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+// Disclaimer https://github.com/hats-finance/hats-contracts/blob/main/DISCLAIMER.md
+
+pragma solidity 0.8.6;
+
+
+import "openzeppelin-solidity/contracts/governance/TimelockController.sol";
+import "./HATVaults.sol";
+
+
+contract HATTimelockController is TimelockController {
+    HATVaults public hatVaults;
+
+    constructor(
+        uint256 _minDelay,
+        address[] memory _proposers,
+        address[] memory _executors
+    // solhint-disable-next-line func-visibility, no-empty-blocks
+    ) TimelockController(_minDelay, _proposers, _executors) {
+    }
+
+    function initializeHATVaults(HATVaults _hatVaults) external onlyRole(EXECUTOR_ROLE) {
```

Comment:
why executor role? 
So I am not sure how this contract will be deployed, but the current suggestion in https://docs.google.com/document/d/1raskPjnNeHQMaOfAgCIzgv7NTTH5uRd2ZBj0yMhSMWM/edit# is to give anyone the executor role, which would not be good. 
Also: is it necessary to be able to reset this? I guess that if we upgrade the hatvaults contract, you would also deploy a new timelockcontroller. Mb just remove this function